cypress/support/Train.js

```diff
@@ -126,6 +126,16 @@ export function SelectAction(expectedResponse, lastResponse)
   })
 }
 
+export function SelectSessionAction(expectedResponse, lastResponse)
```

Comment:
I would prefer that the name of this method AND the data_testid BOTH use End in the name, ie…"SelectEndSessionAction" and "action-scorer-end-session-response". The goal is for variable and function names to clearly match the UI elements they represent and manipulate.

Reply:
While agreed, I wrote the code to supports any adaptive card - e.g. end session, api and form type actions so I opted for the broader adaptive card type chats. The hard coded "end session" code disappears, to be passed as the arg so double win